README.md

```diff
@@ -1 +1,13 @@
-# angular-create-a-directive-lab
\ No newline at end of file
+# Create a custom Directive structure
+
+## Objectives
+
+- Write a Directive and definition Object
+
+## Instructions
+
+Now that we've got the know-how on how to create very basic directives, let's create some.
+
+We're going to have a success directive and an error directive. These should have templates inside them that have a message for when something succeeds, and when something fails. This means we can use the same language everywhere in our app just be referencing these two directives.
+
+They should be named `successMessage` and `errorMessage` so we can use them in Angular via `<success-message></success-message>` and `<error-message></error-message>`.
```

Comment:
Let's be specific here and tell the reader exactly what the template should be for these. An example instruction: "The success message template should render a paragraph tag with the text 'Success!' "